app/assets/stylesheets/layouts/river.scss

```diff
@@ -1,35 +1,118 @@
+
+$bg_color_post_character: #929292;
+$bg_color_post_screenname: #A5A5A5;
+$bg_color_post_author: #BBBBBB;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in // TODO: Should these flip on other themes?
+
+.post-character { background-color: $bg_color_post_character; }
+.post-screenname { background-color: $bg_color_post_screenname; }
+.post-author { background-color: $bg_color_post_author; }
+
+$bg_color_sub: #BDBDBD; // secondary content header
+.sub { background-color: $bg_color_sub; }
 
 /* Rest of Milky River */
-#logo { background-color: #434D73; }
-#nav-top, #user-info { background-color: #212135; }
-#header, #nav-bottom { background-color: #ADB5B9; }
-.post-expander { background-color: #A4B0BD; }
-.pagination .current { background: #718EB9; }
-.post-icon, #current-icon-dropdown, #reply-icon-selector, .replace-icon, .icons-box, .icon-credit { background-color: #A9B4BB; }
-a { color: #3F5970; }
-a:visited { color: #284150; }
-.flash.breadcrumbs, .form-table-ender { background-color: #A3A8A9; }
-.subber { background-color: #B9B6AD; }
+$bg_color_header_logo: #212135; // top header strip for logo
+$bg_color_header_navbottom: #ADB5B9;
+$bg_color_header_middle: #212135; // middle header strip with user items
+$bg_color_header_bottom: #ADB5B9; // bottom header strip with navigation
+
+#header { background-color: $bg_color_header_bottom; }
+#logo { background-color: $bg_color_header_logo; }
+#nav-bottom { background-color: $bg_color_header_navbottom; }
+#nav-top, #user-info { background-color: $bg_color_header_middle; }
+
+/* - links */
+$font_color_link: #3F5970; // link
+$font_color_link_visited: #284150;
+//$font_color_link_hover: #FCFEFD; // link (hovered)
+a { color: $font_color_link; }
+a:visited { color: $font_color_link_visited; }
+
+$bg_color_flash_breadcrumbs: #A3A8A9; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+.flash.breadcrumbs { background-color: $bg_color_flash_breadcrumbs; }
+
+$bg_color_expander: #A4B0BD; // expander on posts
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_page_current: #718EB9;
+.pagination .current { background-color: $bg_color_page_current; }
+
+$bg_color_post_icon: #A9B4BB;
+$bg_color_icon_selector: #A9B4BB;
+//$font_color_icon_selector: #111111;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; }
+
+/* - icons */
+//$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #A9B4BB;
+$bg_color_icon_credit: #A9B4BB;
+
+.replace-icon, .icons-box  { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+$bg_color_subber1: #A3A8A9; // tertiary content header
+$bg_color_subber2: #B9B6AD; // tertiary content header // TODO: EXAMINE THIS
```

Comment:
there's `$bg_color_ender` which should be used for the `.form-table-ender` and which is usually the same as `.subber` but not always – you should use that name for that one instead and there's no need to further examine it, I think:

https://github.com/Marri/glowfic/blob/a27e7e018a85d856a28e33546fecc03b6758bbd9/app/assets/stylesheets/_variables.scss#L96

Reply:
oh is THAT what that does why is it not with the other stuff

Reply:
Which other stuff?

The distinction is that subbers are a type of header, so they're used to present information to people in Big Obvious Ways, whereas the `form-table-ender` is that colored background thing on the last row of a table where you have a "Submit" button. It used to always be styled the same as subber, which is also used for the table headers ("Username", "Password", "Secret", for example), but then one of the layouts wanted to change it, and I realized that actually they were semantically different.